src/System.Reflection.MetadataLoadContext/pkg/System.Reflection.MetadataLoadContext.pkgproj

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <ItemGroup>
+    <ProjectReference Include="..\src\System.Reflection.MetadataLoadContext.csproj" />
```

Comment:
You should add the reference here as well. For that add the following:

```xml
<ProjectReference Include="..\ref\System.ReflectionMetadataLoadContext.csproj">
  <SupportedFrameworks>netstandard</SupportedFrameworks>
</ProjectReference>
```

Reply:
done

Reply:
In order to prevent package errors like
`Packaging.targets(1126,5): error : System.Reflection.MetadataLoadContext should not be supported on .NETFramework,Version=v4.6.1 but has both compile and runtime assets`
I needed:
```xml
<SupportedFramework>uap10.0.16299;net461;netcoreapp2.0;$(AllXamarinFrameworks</SupportedFramework>
```